Model/Methods/GooglePay.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ *
+ * Adyen Payment Module
+ *
+ * Copyright (c) 2023 Adyen N.V.
+ * This file is open source and available under the MIT license.
+ * See the LICENSE file for more info.
+ *
+ * Author: Adyen <magento@adyen.com>
+ */
+namespace Adyen\Payment\Model\Methods;
+
+use Adyen\Payment\Helper\PaymentMethods\PaymentMethodInterface;
+use Adyen\Payment\Model\AdyenPaymentMethod;
+
+class GooglePay extends AdyenPaymentMethod implements PaymentMethodInterface
+{
+    const CODE = 'adyen_googlepay';
+    const TX_VARIANT = 'googlepay';
```

Comment:
Is there a way to support `paywithgoogle`?

Reply:
We could somehow make it work, but since we are already asking merchants to use `googlepay` from v8 upwards, I don't think it's necessary